### PR TITLE
#51 Clippy Fix 5

### PR DESCRIPTION
**Pull request by MrWWheat**
Ignore unused codes.
Split map and count.
Skip to fix too many arguments.

### CODE REVIEW

File: async_fuse/src/protocol.rs [hunk 1]
```diff
@@ -424,6 +424,7 @@ pub enum FuseNotifyCode {
 }
 
 // The read buffer is required to be at least 8k, but may be much larger
+#[allow(dead_code)]
```
Comment (pwang7) [inline async_fuse/src/protocol.rs:427]: for unused const variables, add a mod to group them, and allow dead_code for the mod.

File: fuse_ll/src/memfs.rs [hunk 2]
```diff
@@ -4,6 +4,7 @@ use crate::fuse::{
 };
 use libc::{EEXIST, EINVAL, ENODATA, ENOENT, ENOTEMPTY};
 use log::{debug, error}; // info, warn
+use nix::dir::Entry;
 use nix::dir::{Dir, Type};
```
Comment (pwang7) [inline fuse_ll/src/memfs.rs:8]: use nix::dir::{Dir, Entry, Type};

Comment (MrWWheat) [reply fuse_ll/src/memfs.rs]: done

File: async_fuse/src/protocol.rs (hunk 1 above)
Comment (MrWWheat) [reply async_fuse/src/protocol.rs:481]: done

File: async_fuse/src/protocol.rs [hunk 3]
```diff
@@ -478,8 +478,15 @@ pub enum FuseNotifyCode {
     FUSE_NOTIFY_CODE_MAX,
 }
 
-// The read buffer is required to be at least 8k, but may be much larger
-pub const FUSE_MIN_READ_BUFFER: usize = 8192;
+#[allow(dead_code)]
+pub mod fuse_configs {
```
Comment (pwang7) [inline async_fuse/src/protocol.rs:482]: how about group more related const variables and use a more meaningful mod name?

Comment (MrWWheat) [reply async_fuse/src/protocol.rs]: done.

**Pull request merged**